code/modules/holiday/spacemas.dm

```diff
@@ -1274,3 +1288,28 @@ var/static/list/santa_snacks = list(/obj/item/reagent_containers/food/drinks/egg
 	icon_state = "xmascrate"
 	icon_opened = "xmascrateopen"
 	icon_closed = "xmascrate"
+
+/obj/xmas_ephemeral_turf
+	icon = 'icons/turf/floors.dmi'
+	icon_state = "floor"
```

Comment:
i don't like this icon, it's just going to cover the other one and look weird

also this specific turf spawning object is a bad idea imo